cromwell-drs-localizer/src/main/scala/drs/localizer/DrsLocalizerMain.scala

```diff
@@ -42,47 +43,91 @@ object DrsLocalizerMain extends IOApp with StrictLogging {
     initialInterval = 10 seconds, maxInterval = 60 seconds, multiplier = 2)
 
   val defaultDownloaderFactory: DownloaderFactory = new DownloaderFactory {
-    override def buildAccessUrlDownloader(accessUrl: AccessUrl, downloadLoc: String, hashes: Hashes): IO[Downloader] =
-      IO.pure(AccessUrlDownloader(accessUrl, downloadLoc, hashes))
-
     override def buildGcsUriDownloader(gcsPath: String, serviceAccountJsonOption: Option[String], downloadLoc: String, requesterPaysProjectOption: Option[String]): IO[Downloader] =
       IO.pure(GcsUriDownloader(gcsPath, serviceAccountJsonOption, downloadLoc, requesterPaysProjectOption))
+
+    override def buildBulkAccessUrlDownloader(urlsToDownload: List[ResolvedDrsUrl]): IO[Downloader] = {
+      IO.pure(BulkAccessUrlDownloader(urlsToDownload))
```

Comment:
Why is this an `IO`?  Presumably to make something up-stack happier, since this is just instantiating a case class? I would move the `IO.pure` wrapper up to the place its needed (same for `GcsUriDownloader`).